### DISCUSSION TITLE
Apa si Hacktoberfestitu ?

### DISCUSSION THREAD

**Discussion by dedesyarif100** [troke12/jumlah-angka#5] [Q&A] [answered]
Apa si Hacktoberfestitu ?

**Comment by troke12** [comment, marked as answer]
Itu lho mas acara dari digitalocean.com

**Comment by billymaulana** [comment]
HACKTOBERFEST IS DIGITALOCEAN’S ANNUAL EVENT THAT ENCOURAGES PEOPLE TO CONTRIBUTE TO OPEN SOURCE THROUGHOUT OCTOBER. MUCH OF MODERN TECH INFRASTRUCTURE—INCLUDING SOME OF DIGITALOCEAN’S OWN PRODUCTS—RELIES ON OPEN-SOURCE PROJECTS BUILT AND MAINTAINED BY PASSIONATE PEOPLE WHO OFTEN DON’T HAVE THE STAFF OR BUDGETS TO DO MUCH MORE THAN KEEP THE PROJECT ALIVE. HACKTOBERFEST IS ALL ABOUT GIVING BACK TO THOSE PROJECTS, SHARPENING SKILLS, AND CELEBRATING ALL THINGS OPEN SOURCE, ESPECIALLY THE PEOPLE THAT MAKE OPEN SOURCE SO SPECIAL.

FOR THE PAST 9 YEARS, THOUSANDS OF PEOPLE—CODERS AND NON-CODERS ALIKE—HAVE PARTICIPATED IN HACKTOBERFEST TO SUPPORT THE PROJECTS THEY USE AND LOVE, LEARN AND PRACTICE SKILLS THAT WILL ENHANCE THEIR CAREERS, AND MEET NEW PEOPLE WHO LOVE OPEN SOURCE AS MUCH AS THEY DO.
